src/puppetlabs/puppetdb/query/reports.clj

```diff
@@ -84,13 +83,10 @@
   and count (if supplied)."
   [version url-prefix query-sql]
   {:pre [(map? query-sql)]}
-  (let [{[sql & params] :results-query
-         count-query    :count-query} query-sql
-         result {:result (query/streamed-query-result
-                          version sql params
-                          ;; The doall simply forces the seq to be traversed
-                          ;; fully.
-                          (comp doall (munge-result-rows version url-prefix)))}]
+  (let [{:keys [count-query results-query]} query-sql
```

Comment:
They are the old query-eng fn's afaik, old code that needs to stick around as long as some of our old endpoints are alive